src/main/java/edu/ucla/library/services/utility/ExitCodes.java

```diff
@@ -0,0 +1,32 @@
+
+package edu.ucla.library.services.utility;
```

Comment:
I'm curious how you see other utilities being added to this repo. The ending `utility` package name sounds singular. Would it be better to use `utils.props` or something (so there could be a `utils.somethingelse` in the future)? Or how do you see adding additional utilities?

Reply:
Right... wasn't thinking about how to extend/add to package.

Reply:
On this, I also kind of wonder about the general utilities idea. In general it makes sense, but for this particular use case we're putting this jar into our Docker image. I'm wondering if it wouldn't be better to have just this single utility so that we keep the size of the jar down -- reducing the size of the container. Thoughts?

Reply:
Yes, we could have a build profile that included certain things in the jar, but I'm not sure how that would work with pushing the Jar to Maven Central. You might be able to use Maven classifiers to have a jar that's built with everything and then one that's just built with this utility(?) Maven uses classifiers to create Jars of Javadocs that get uploaded with the main project Jar and they are sometimes used for OS specific bundles of code that get published with the main Jar. I'm not sure if this is abusing the classifiers idea though.

Reply:
If we keep this project/repo to just the props builder, how much of a pain is it to rename the repo (e.g., to props-builder) to keep the services-utilities name available if we decide to build a broader utilities package?

Reply:
Renaming the repo is pretty straight-forward. GitHub will keep redirecting from services_utilities to whatever we rename it to so the existing Codacy/Snyk/Travis setups should continue to work (if we didn't want to tackle updating them yet). Since the new repo/name would be services-utilities I don't think that would conflict (I believe GitHub will treat the underscore vs. hyphen as a significant difference).

Reply:
We can rename on the next ticket that will publish to Maven Central.